backend/tracim_backend/models/social.py

```diff
@@ -17,6 +17,7 @@ class UserFollower(DeclarativeBase):
     follower = relationship("User", remote_side=[User.user_id], foreign_keys=[follower_id])
     leader_id = Column(Integer, ForeignKey("users.user_id"), nullable=False, primary_key=True)
     leader = relationship("User", remote_side=[User.user_id], foreign_keys=[leader_id])
+    # TODO - G.M - 2021-03-10:  use CreationDateMixin instead
```

Comment:
Did you decide against using CreationDateMixin now? Or did you forget to do it?

Reply:
- The field name are not the same as for the mixin
- i decided to not change too much unrelated stuff in this PR. I should probably either create a new issue or fix it now.